check_stability.py

```diff
@@ -283,6 +286,17 @@ def wptrunner_args(self, root):
             "test_types": ["testharness", "reftest"]
         }
 
+    def extra_setup(self):
+        # https://bugs.chromium.org/p/chromium/issues/detail?id=713947
+        logger.debug("DBUS_SESSION_BUS_ADDRESS %s" % os.environ.get("DBUS_SESSION_BUS_ADDRESS"))
+        if "DBUS_SESSION_BUS_ADDRESS" not in os.environ:
```

Comment:
Is that necessary given that there are no other environements at the moment and everything will certainly break in !linux, given that we are downloading linux-specific binaries?